src/framework/dropdown/DropdownGroup.jsx

```diff
@@ -0,0 +1,41 @@
+
+import React, {
+  PropTypes,
+} from 'react';
+import classNames from 'classnames';
+
+import Dropdown from './Dropdown.jsx';
+
+const DropdownGroup = (props) => {
+  const optionToList = props.options.reduce(
+    (accumulator, optionList, currentIndex) => (
+      accumulator.concat(optionList.map((option, index) => (
+        Object.assign({}, option, {
+          class: classNames(props.optionsClass[currentIndex], {
+            dropdownGroupFirst: (accumulator.length > 0 && index === 0),
+          }),
+        })
+      )))
+    ),
+  []);
+
+  const extendedProps = Object.assign({}, props, {
+    options: optionToList,
+  });
+
+  return (
+    <Dropdown
+      {...extendedProps}
+    />
+  );
+};
+
+DropdownGroup.propTypes = Object.assign({}, Dropdown.propTypes, {
+  optionsClass: PropTypes.array,
+});
+
+DropdownGroup.defaultProps = Object.assign({}, Dropdown.defaultProps, {
+  classes: 'dropdownGroup',
+});
+
+export default DropdownGroup;
```

Comment:
Still think that `DropdownGroup` could be misleading. It rather indicates that it's a group of dropdowns. Perhaps in direction of `GroupedOptionsDropdown`?! We have a `ButtonGroup` which is a group of buttons (specifically `GroupedButton`s which is perhaps also not perfect wording).

Reply:
I don't think it is misleading because in the rare case that ui framework will need to provide a component that provides a group of dropdowns it will be name dropdowns as plural